icicle-compiler/test/Icicle/Test/Sea/Name.hs

```diff
@@ -27,14 +29,32 @@ prop_mangle_works =
     isNothing (asSeaName text1) ==>
     isJust . asSeaName . takeSeaName . mangleToSeaName $ text1
 
-prop_mangle_unique :: Property
-prop_mangle_unique =
-  forAll genValidUtf8 $ \text1 ->
-  forAll genValidUtf8 $ \text2 ->
-  mangleToSeaName text1 == mangleToSeaName text2 ==>
-  text1 == text2
```

Comment:
This test wasn't actually doing anything, I put a `trace` / `classify` in and found that the only successful test cases were the empty string.